security/auto-host-endpoints.md

```diff
@@ -0,0 +1,160 @@
+---
+title: Protect Kubernetes nodes
+description: Protect Kubernetes nodes with host endpoints managed by {{site.prodname}}
+---
+
+### Big picture
+
+Secure Kubernetes nodes with host endpoints managed by {{site.prodname}}.
+
+### Value
+
+{{site.prodname}} can automatically create host endpoints for your Kubernetes nodes. The lifecycle of these host endpoints are managed by {{site.prodname}} in response to an evolving cluster to ensure policy protecting your nodes is always enforced.
```

Comment:
{{site.prodname}} can automatically create host endpoints for your Kubernetes nodes. This means {{site.prodname}} can manage the lifecycle of host endpoints  as your cluster evolves, ensuring nodes are always protected by policy.